src/main/resources/locales/bill/crown-firm-list.json

```diff
@@ -0,0 +1,47 @@
+{
+  "title": "",
+  "headerPart1": "",
+  "headerPart2": "Rhestr Ddyddiol",
+  "listDate": "List for/Rhestr ar gyfer",
+  "listUpdated": "Last Updated/Diweddarwyd diwethaf DATE at/yn ",
```

Comment:
Missing colon after Last Updated/Diweddarwyd

Reply:
This has now been fixed.